docroot/themes/humsci/humsci_airy/scss/_horizontal-card.scss

```diff
@@ -0,0 +1,10 @@
+//Able to import base styles/mixins/variables from humsci_basic here
+@import '../../humsci_basic/scss/base.scss';
```

Comment:
While working on the Decanter import card, I discovered how import paths work, so that this import would not have to be relative like this. Just an FYI though. Nothing actionable.